qiskit/qpy/__init__.py

```diff
@@ -128,6 +128,30 @@
 by ``num_circuits`` in the file header). There is no padding between the
 circuits in the data.
 
+.. _qpy_version_10:
+
+Version 10
+==========
+
+Version 10 adds support for new fields in the :class:`~.TranspileLayout` class added in the Qiskit
+0.45.0 release. The ``LAYOUT`` struct is updated to have an additional ``input_qubit_count`` field.
+WIth version 10 the ``LAYOUT`` struct is now:
```

Comment:
```suggestion
With version 10 the ``LAYOUT`` struct is now:
```

(though tbh we're probably going to have to resolve some merge conflicts with a couple of other PRs that are also make QPY version 10s)

Reply:
Yeah, we can handle merge conflicts as they come up. Having >1 change in version 10 is fine I think